ui/app/components/app/connected-accounts-list/connected-accounts-list-item/connected-accounts-list-item.component.js

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import Identicon from '../../../ui/identicon'
+
+export default class ConnectedAccountsListItem extends PureComponent {
```

Comment:
[Q] Learnin' patterns and shiz... why did you opt to make this a class-based component over a functional one?

Reply:
Habit & I had written most of this over a month ago 😅

Reply:
We should standardize on functional components going forward